bin/run-ted.sh

```diff
@@ -1,25 +1,15 @@
 #!/bin/sh
-
 set -xe
 
-export ds_importer="ted"
-
-export ds_train_batch_size=16
-export ds_dev_batch_size=8
-export ds_test_batch_size=8
-
-export ds_learning_rate=0.0001
-
-export ds_epochs=50
-export ds_validation_step=10
-export ds_display_step=10
-
-export ds_dropout_rate=0.30
-export ds_checkpoint_step=1
-
 if [ ! -f DeepSpeech.py ]; then
     echo "Please make sure you run this from DeepSpeech's top level directory."
     exit 1
 fi;
 
-python -u DeepSpeech.py
+python -u DeepSpeech.py \
+  --importer ted \
+  --train_batch_size 16 \
+  --dev_batch_size 8 \
+  --test_batch_size 8 \
+  --learning_rate 0.0001 \
+  "$@"
```

Comment:
Previous values for ds_epochs, ds_validation_step, ds_display_step,  ds_dropout_rate, and a checkpoint interval should be passed in here.